src/OnlySpans.PolyLeads.Api/Data/Entities/PermittedSource.configuration.cs

```diff
@@ -12,11 +12,11 @@ public void Configure(EntityTypeBuilder<PermittedSource> builder)
            .HasMaxLength(256);
         
         builder.HasData(
-            new  {Id = 1, BaseUrl = new Uri("https://www.spbstu.ru"), Description = "Сайт политеха"},
-            new  {Id = 2, BaseUrl = new Uri("https://college.spbstu.ru"), Description = "Институт среднего профессионального образования СПБПУ"},
-            new  {Id = 3, BaseUrl = new Uri("https://enroll.spbstu.ru"), Description = "Приемная комиссия СПБПУ"},
-            new  {Id = 3, BaseUrl = new Uri("https://iccs.spbstu.ru/"), Description = "ИКНК СПБПУ"},
-            new  {Id = 3, BaseUrl = new Uri("https://physmech.spbstu.ru/"), Description = "ФизМех СПБПУ"}
+            new  {Id = (long) 1, BaseUrl = new Uri("https://www.spbstu.ru"), Description = "Сайт политеха"},
```

Comment:
```suggestion
            new()  {Id = 1, BaseUrl = new Uri("https://www.spbstu.ru"), Description = "Сайт политеха"},
```

или 

```suggestion
            new PermittedSource  {Id = 1, BaseUrl = new Uri("https://www.spbstu.ru"), Description = "Сайт политеха"},
```

тогда (long) надо будет

Reply:
не забудь миграцию сделать после этого

Reply:
а хотя хуй с ним